ckanext/subscribe/controller.py

```diff
@@ -70,9 +72,11 @@ def signup(self):
                             'administrator for help'))
             return self._redirect_back_to_subscribe_page_from_request(data_dict)
         else:
+            subscribe_title = dataset_title or group_title
             h.flash_success(
-                _('Subscription requested. Please confirm, by clicking in the '
-                  'link in the email just sent to you'))
+                _('Subscription to {} was successful, please confirm '
+                    'your subscription by checking your email inbox and spam/trash folder'.format(subscribe_title)))
```

Comment:
Please fix the indentation on this line.

Reply:
Also should the message be
_by checking your email inbox **or** spam/trash folder_?